contrib/test/integration/main.yml

```diff
@@ -21,15 +21,20 @@
       vars:
         cri_tools_git_version: "b42fc3f364dd48f649d55926c34492beeb9b2e99"
 
+    - set_fact:
+        force: "{{ force_clone }}"
+
     - name: clone build and install kubernetes
       include: "build/kubernetes.yml"
       vars:
         k8s_git_version: "cri-o-node-e2e-patched-logs"
         k8s_github_fork: "runcom"
+        force_clone: "{{ force }}"
```

Comment:
I'm not clear on the format here, so take this with a grain of salt, but going from `force_clone` to `force` above and then from `force` back to `force_clone` here seems confusing.  Is it possible to stick with `force_clone` throughout?

Reply:
if you use `force_clone: "{{ force_clone }}"` then Ansible will crash in an endless recursion, as it tries to read its value from the same block.

I am not sure if a set done before this block would help to maintain the same name

Reply:
> if you use `force_clone: "{{ force_clone }}"` then Ansible will crash in an endless recursion

Can you just leave `force_clone` off of this `include` (and skip the `set_fact` too) and have it be [inherited][1] through?  I'm not familiar with Ansible's variable inheritance, but from the docs it looks like that might work.

[1]: http://docs.ansible.com/ansible/latest/playbooks_variables.html#variable-precedence-where-should-i-put-a-variable

Reply:
I've dropped this patch from the PR for now and see if I can rewrite it in a different way if needed.